src/main.rs

```diff
@@ -434,6 +437,10 @@ impl Config {
         cmd.arg("--target-dir").arg(&self.target_dir);
         cmd.arg("--manifest-path").arg(&self.pkg.manifest_path);
 
+        if let Some(jobs) = self.cli.jobs {
+            cmd.arg("--jobs").arg(jobs.to_string());
```

Comment:
```suggestion
            cmd.arg("--jobs").arg(jobs);
```

Reply:
`error[E0277]: the trait bound `u32: std::convert::AsRef<std::ffi::OsStr>` is not satisfied`
I've tried that before. Only works if I change `jobs: Option<u32>` to `jobs: Option<String>`

Reply:
You are right :)